slowquant/qiskit_interface/linear_response/naive.py

```diff
@@ -30,7 +30,7 @@
 class quantumLR(quantumLRBaseClass):
     def run(
         self,
-        do_rdm: bool = False,
+        do_rdm: bool = True,
```

Comment:
It is fine for now to have True. 
But I think once we would understand a bit more what the implications of RDMs on shot noise is, we might revisit it :) 
What I also like to add is that by having default True here, means that it is by default a different approach than allproj that cannot use RDMs.